jsgui/src/main/webapp/index.html

```diff
@@ -48,6 +51,7 @@
             <div class="nav-collapse menubar-top">
                 <ul class="nav">
                     <li><a href="#/v1/home" class="nav1 nav1_home">Home</a></li>
+                    <li><a href="#v1/editor/" class="nav1 nav1_editor">Composer</a></li>
```

Comment:
@neykov @Graeme-Miller so _all_ the links here should be changed to look like the editor one?